src/components/projects-summary/projects-summary.html

```diff
@@ -1,11 +1,75 @@
 <div class="catalogs-projects-summary-panel">
-  <button class="create-button btn btn-primary">
+  <button ng-if="$ctrl.canCreate" class="create-button btn btn-primary" ng-click="$ctrl.openNewProjectPanel()">
     <span class="fa fa-plus"></span>
     <span class="create-button-text">Create Project</span>
   </button>
-  <h2>Getting Started</h2>
-  <div class="getting-started-panel">
-    {{$ctrl.gettingStartedText}}
+  <div class="catalogs-create-project-panel" ng-if="$ctrl.showNewProjectPanel">
+    <div class="arrow"></div>
+    <div class="catalogs-create-project-title">
+      <span>Create Project</span>
+      <a class="pull-right" ng-click="$ctrl.closeNewProjectPanel()">
+        <span class="catalogs-create-project-close pficon pficon-close"></span>
+      </a>
+    </div>
+    <create-project alerts="$ctrl.alerts" is-dialog="true" redirect-action="$ctrl.onNewProject" on-cancel="$ctrl.closeNewProjectPanel"></create-project>
+  </div>
+
+  <div ng-if="$ctrl.showGetStarted">
+    <h2>Getting Started</h2>
+    <div class="getting-started-panel">
+      {{$ctrl.gettingStartedText}}
+    </div>
+  </div>
+  <div ng-if="!$ctrl.showGetStarted" class="catalogs-project-summary-list">
+    <div ng-if="$ctrl.maxDisplayProjectsReached" class="projects-count-area">
+      <span class="counter-text">{{$ctrl.maxDisplayProjects}}</span>
+      <span>of</span>
+      <span class="counter-text">{{$ctrl.totalProjects}}</span>
+      <span>Projects</span>
+      <a href="#" class="projects-view-all" ng-click="$ctrl.showAllProjects()">View All</a>
```

Comment:
Can't this be `href=""` here and below? Then we're not adding or changing the URL hash.

https://docs.angularjs.org/api/ng/directive/a

> Modifies the default behavior of the html a tag so that the default action is prevented when the href attribute is empty.